src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml

```diff
@@ -4224,6 +4299,37 @@ components:
         is_locked:
           type: boolean
 
+    StateAccountLockerPageAccountResourcesRequest:
+      allOf:
+        - $ref: "#/components/schemas/AtLedgerStateMixin"
+        - $ref: "#/components/schemas/CursorLimitMixin"
+        - type: object
+          required:
+            - account_locker_address
+            - account_address
+          properties:
+            account_locker_address:
+              $ref: "#/components/schemas/Address"
+            account_address:
+              $ref: "#/components/schemas/Address"
```

Comment:
We're missing optional `resource_addresses` (and I assume a good description of the two modes is to come on the endpoint itself)